lib/cdo/workshop_constants.rb

```diff
@@ -0,0 +1,35 @@
+module WorkshopConstants
```

Comment:
It is used here: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/ops-dashboard/index.haml#L73-L94

:(  I didn't turn this up in my earlier search and it didn't fail any rake tests because it's in the javascript. Do we have tests for that?

So, I'll keep it in shared and update ops_dashboard to use WorkshopConstants